codegen/test_data/clients/bar.gogen

```diff
@@ -542,8 +542,14 @@ func (c *barClient) ArgWithParams(
 	// Generate full URL.
 	fullURL := c.httpClient.BaseURL + "/bar" + "/argWithParams" + "/" + string(r.UUID) + "/segment" + "/" + string(r.Params.UserUUID)
 
+	queryValues := &url.Values{}
```

Comment:
these values are only written into URL params and should not be written into query params.

If the http.ref annotation is params or headers then it must NOT be written into the query parameters.

Reply:
Done